x11-misc/bbmail/bbmail-0.9.3-r1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit autotools
+
+DESCRIPTION="blackbox mail notification"
+HOMEPAGE="https://sourceforge.net/projects/bbtools"
+SRC_URI="mirror://sourceforge/bbtools/${P}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~ppc ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND="x11-wm/blackbox
+	x11-libs/libX11
+	x11-libs/libXext"
+DEPEND="${RDEPEND}
+	x11-proto/xproto"
+
+PATCHES=( "${FILESDIR}"/${P}-gcc4.3.patch
+	"${FILESDIR}"/${P}-gcc4.4.patch )
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_install () {
+	default
+	dobin scripts/bbmailparsefm.pl
```

Comment:
1. this is a perl script, without `RDEPEND`'ing on perl?
2. Please check that the perl script uses PATH lookups, i.e., https://blogs.gentoo.org/mgorny/2016/02/08/a-quick-note-on-portable-shebangs/ (and if you need to fix it, please use a patch instead of a `sed`)